OSVR-Unity/Assets/OSVRUnity/Resources/OsvrDistortion.shader

```diff
@@ -18,66 +18,59 @@
 /// limitations under the License.
 /// </copyright>
 /// <summary>
-/// Based on Unity's (Pro Only) Image Effect: Fisheye.shader
+/// Long ago, based on Unity's (Pro Only) Image Effect: Fisheye.shader
```

Comment:
Yeah, not sure how "based on" it actually is, would have to ask @DuFF14 . I know I basically re-wrote the whole C# part of the code, so that should be fine (and can probably just remove the ref to the unity thing)